kano_updater/scenarios.py

```diff
@@ -593,6 +592,9 @@ def enable_audio_device():
         # tell dashboard to skip Overworld and kit setup onboarding phase
         run_for_every_user('touch ~/.dashboard-click-onboarding-done')
 
+        # tell kano-init to setup to kit to go Dashboard the systemd way
+        run_cmd_log('kano-init finalise --force')
```

Comment:
Huh, how come this guy is still here? Surely we need just one of these in the the postupdate `beta_390_to_beta_3100()` function.